requirements.txt

```diff
@@ -36,6 +36,7 @@ requests[security] >= 2.25.1, < 3.0.0
 tabulate==0.9.0
 zipstream-new==1.1.8
 sentry-sdk[flask]==1.31.0
+prometheus-client==0.17.1
```

Comment:
The latest version on PyPI seems to be `0.19.0`. I assume this is because you started on this a while ago? Not saying we'd need to update immediately.

Reply:
That’s a good point, I’ll bump it to the latest version, saves us a Dependabot PR in the future.

Reply:
Sooo… I pinned the exact version in servicelayer, so of course I cannot simply upgrade it here 🙃. Even though it’s still a 0.x.x, from looking at past releases, it seems that there have been almost no breaking changes so far. So I’m inclined to relax the version constraints in servicelayer to `>= 0.19.0, < 1.0.0`.

As that requires another round of releases and bumping versions, I think I’d prefer doing that separately though, as the latest release doesn’t include any changes relevant to us…

Reply:
Sorry I didn't get back to you on this. I'm fine with the proposed changes in separate PRs. 👍